providers/azure/aci.go

```diff
@@ -51,6 +51,25 @@ type AuthConfig struct {
 	RegistryToken string `json:"registrytoken,omitempty"`
 }
 
+const westEuropeRegion = "westeurope"
+const westUsRegion = "westus"
+const eastUsRegion = "eastus"
+const southEastAsiaRegion = "southeastasia"
+
+// IsValidACIRegion checks to make sure we're using a valid ACI region
+func isValidACIRegion(region string) bool {
+	switch region {
```

Comment:
you need to normalize the region by removing all white spaces, truncated and lowercase before the switch, since "West US" and "WestUS" are also valid Azure region names.